linode/manager
# M3-5928: Upgrade the Material UI Packages

## Description 📝

> **Warning**: Github Actions checks will fail, but that is expected. Next Jira task is to resolve this.

- Starts the MUI v4 to v5 upgrade
- Completes the migration guide provided by MUI
  - The diff is massive because we ran the automated migration script provided by MUI
- Does the minimal amount of work to get the app to load

## How to test 🧪

- Try to `yarn up` the app and verify that things mostly work
  - There will be some UI regressions and Typescript errors
